src/ui/DatePeriodPicker.tsx

```diff
@@ -67,7 +67,7 @@ const DatePeriodPicker = ({
           minDate={
             dateStart && dateStart.getTime() > new Date().getTime()
               ? dateStart
-              : new Date()
+              : minDate
```

Comment:
It's the prop received by the component, it's to avoid that prop being overriden in case that ternary condition fails, otherwise the prop is not respected.